classes/class-wc-connect-nux.php

```diff
@@ -395,36 +392,37 @@ public function show_banner_before_connection() {
 
 			$jetpack_status = $this->get_jetpack_install_status();
 
-			$button_text = __( 'Connect', 'woocommerce-services' );
+			$button_text  = __( 'Connect', 'woocommerce-services' );
+			$banner_title = __( 'Connect your store to activate WooCommerce Services', 'woocommerce-services' );
```

Comment:
This text and title only shows up in the case where Jetpack is active but not connected. Why "Connect Jetpack" for the other cases, but "Connect your store" with no mention of Jetpack in this one? If we're clarifying what it is we're connecting in the other two cases, why not in this one?

Reply:
Good catch - this also highlighted some deficiencies in our copy regarding features - working on that now.